cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -831,24 +884,59 @@ public void removeViewTag(HttpRequest request, HttpResponder responder,
     return toReturn.toArray(new String[toReturn.size()]);
   }
 
-  // *** Search endpoints ***
-
   @GET
   @Path("/namespaces/{namespace-id}/metadata/search")
   public void searchMetadata(HttpRequest request, HttpResponder responder,
                              @PathParam("namespace-id") String namespaceId,
                              @QueryParam("query") String searchQuery,
-                             @QueryParam("target") List<String> targets) throws Exception {
-    Set<MetadataSearchTargetType> types = ImmutableSet.of();
+                             @QueryParam("target") List<String> targets,
+                             @QueryParam("sort") @DefaultValue("") String sort,
```

Comment:
because we want to be able to say "name asc, create_time desc" later. Or even support other formats that may be supported by search backends like Solr later.